internal/util/rand.go

```diff
@@ -4,19 +4,23 @@ import (
 	"math/rand"
 )
 
-func UniqRands(l int, n int) []int {
-	set := make(map[int]struct{})
-	nums := make([]int, 0, l)
-	for {
-		num := rand.Intn(n)
-		if _, ok := set[num]; !ok {
-			set[num] = struct{}{}
-			nums = append(nums, num)
-		}
-		if len(nums) == l {
-			goto exit
-		}
+func UniqRands(count int, choiceCount int) []int {
```

Comment:
This switches the order of the arguments, but the caller still uses the original argument order.

I see you don't like `l` and `n` and that's fair, they're not super clear. But I find "count" being in both of your argument names to be confusing. I think we can come up with better names. Here's a proposal (back in the original order): `(quantity, maxval int)`

Reply:
I accept your opinion, and I will fix it.